### ISSUE TITLE
accessibility issue 30, 32 (low) - remove aria-polite for osd parent element, remove empty labels

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#257]
There is also an aria-live=”polite” on these “buttons”, which is not necessary.

There are two empty form labels within the image view – in the row with the image control buttons.
These should be removed.

**Comment by yulgit1** [comment]
resolved:https://github.com/ycba-cia/blacklight-collections2/commit/50824406a19f3f21b4ad45c89ec77c0b46ecd527
